python/feathub/feature_views/feature_view.py

```diff
@@ -133,25 +133,35 @@ def get_output_fields(self, source_fields: List[str]) -> List[str]:
             raise FeathubException(
                 "Build this feature view before getting output fields."
             )
-        output_fields = []
+
+        feature_fields = [feature.name for feature in self.get_resolved_features()]
```

Comment:
Can you update the Python doc to explicitly describe the order of fields emitted by the FeatureView?

And should we put features explicitly specified by users before features that are implicitly included (e.g. keys, timestsamp)? This approach allows users to easily determine the positions of features that are explicitly specified.

Reply:
I updated the Python doc.

I thought about putting features before reserved fields, but users are more accustomed to having the key fields before others. And users do not access fields by positions, right? If users really need a specific position, they can add a DerivedFeatureView to select all fields in their order.

Reply:
Position of fields matters here because otherwise we won't need this PR.

The concern with putting keys before user-specified features is that users won't have full control of the order of fields emitted by the FeatureView.

Using an extra DerivedFeatureView is more complicated than using the existing FeatureView to get the expected orders.

Reply:
According to the offline discussion, we agree that the order of keys and features doesn't matter currently. But the docs need to be clear.

I've updated the Python doc again, please have a check.